ui/radar/monsters.js

```diff
@@ -0,0 +1,787 @@
+'use strict';
+
+let Monsters = {
+  'Vogaal Ja': {
+    'name': {
+      'en': 'Vogaal Ja',
+      'cn': '丑男子 沃迦加',
+      'ja': '醜男のヴォガージャ',
+    },
+    'hp': 100000,
```

Comment:
It seems like everything has this hp value.  Would it make more sense to have a default minimum hp in code somewhere rather than specifying it here?

Reply:
No they have different hp values, we use this to determine whether a player can be detected if he has the same name as the hunt mob (which only happens in cn server since we don't have first/last names?).
I'm just lazy to dig out the true value of hp so I just add some lower bound of mob hp. For 2.0, it's 100000 for the A and 10000000 for the S and for 3.0+, 10000000 for the S and A.
It's kinda difficult to dig out all those HPs, maybe NotoriousMonster can help, as you said below.

Reply:
I think you could use the ids of the monsters (which always start with `4` instead of `1` for players) to differentiate whether a mob is a player or not, rather than using hp.

Reply:
I think Chocobo also starts with `4`, just like the monsters. And Chinese players can name their Chocobo names to hunt mobs lol.